src/Nethermind/Nethermind.Core/IKeyValueStore.cs

```diff
@@ -69,6 +69,11 @@ public interface IWriteOnlyKeyValueStore
         void Remove(ReadOnlySpan<byte> key) => Set(key, null);
     }
 
+    public interface IMergeableKeyValueStore
```

Comment:
Shouldn't it be?
```suggestion
    public interface IMergeableKeyValueStore : IKeyValueStore
```

or `ISortedKeyValueStore`?